rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -73,6 +73,7 @@ rabbitmq_ulimit: 65535
 # since memory utilization is primarily driven by the objects placed in memcached.
 memcached_connections: 16384
 
+#TODO (evrardjp): Remove this when ocata
 # Keystone overrides
 keystone_token_driver: "sql"
```

Comment:
OSA was doing something different, if not fernet (read: UUID nowadays) it was using memcache as backend. Here RPC forced to have sql persistence instead. 

I'd rather move to fernet tokens, and keep this as long as we have a fallback to uuid, because we (knowingly) already took a different path than OSA.